modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -191,8 +199,10 @@ class SeqTranslate(site: Site, systems: Systems, settings: Settings) {
 
     val configs = sequence.config.getAllSteps.toList
 
+    val nextToRun = configs.map(extractStatus).lastIndexWhere(s => s === StepState.Completed || s === StepState.Skipped) + 1
```

Comment:
i guess you can use `isFinished` here too

Reply:
No, because `isFinished` applies to `Step`.